src/App.tsx

```diff
@@ -1,10 +1,23 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoError, TodoFooter, TodoList } from './components';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+const USER_ID = 10624;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | null>(null);
+  const [filteringMode, setFilteringMode] = useState<string>('all');
+  const [error, setError] = useState<string | null>(null);
```

Comment:
Simple types can be infered
```suggestion
  const [filteringMode, setFilteringMode] = useState('all');
  const [error, setError] = useState('');
```